src/controller/python/README.md

```diff
@@ -1,244 +1,255 @@
-	UP RUNNING
-	RX bytes:0 acl:95 sco:0 events:205 errors:0
-	TX bytes:2691 acl:95 sco:0 commands:98 errors:0
+   ```
+   sudo out/python_env/bin/chip-device-ctrl --bluetooth-adapter=hci2
+   ```
 
-hci1:	Type: Primary  Bus: Virtual
-	BD Address: 00:AA:01:00:00:23  ACL MTU: 192:1  SCO MTU: 0:0
-	UP RUNNING
-	RX bytes:0 acl:95 sco:0 events:208 errors:0
-	TX bytes:3488 acl:95 sco:0 commands:110 errors:0
-```
+## Working with Python CHIP Controller
 
-Then you can choose the adapter to use in command line arguments of the device
-controller:
+### 1. Prepare the Matter accessory.
 
-For example, add `--bluetooth-adapter=hci2` to use the virtual interface `hci2`
-listed above.
+In this tutorial, the [Matter: Light Bulb](/examples/lighting-app) example with Bluetooth LE commissioning will be used, but similar steps can be executed with the rest of the examples. Check [Matter examples](/examples) for the list of available samples.
 
-```
-chip-device-ctrl --bluetooth-adapter=hci2
-```
+Build and program the device with the Matter accessory firmware by following the example's documentation.
 
-## Usage / BLE Secure Session Establishment
+### 2. Enable Bluetooth LE advertising on Matter accessory device.
 
-1. Run CHIP Device Controller
+Some examples are configured to advertise automatically on boot, others require physical trigger like pushing the button. Follow the Matter accessory example's documentation to learn how Bluetooth LE advertising is enabled.
 
-> Running as root via `sudo` to ensure permissions to interface with the
-> bluetooth adapter.
+### 3. Discover Matter accessory device over Bluetooth LE
+
+An uncommissioned accessory device advertises over Bluetooth LE. Run the following command to scan all advertised Matter devices:
 
 ```
-sudo chip-device-ctrl
+chip-device-ctrl > ble-scan
 ```
 
-or select the bluetooth interface by command line arguments.
+### 4. Connect to Matter accessory device over Bluetooth LE
+
+The controller uses a 12-bit value called **discriminator** to discern between multiple commissionable device advertisements. Moreover, a 27-bit **PIN code** is used by the controller to authenticate in the device. You can find those values in the logging terminal of the device (e.g. UART), as listed below.
 
 ```
-sudo chip-device-ctrl --bluetooth-adapter=hci2
+I: 254 [DL]Device Configuration:
+I: 257 [DL] Serial Number: TEST_SN
+I: 260 [DL] Vendor Id: 9050 (0x235A)
+I: 263 [DL] Product Id: 20043 (0x4E4B)
+I: 267 [DL] Product Revision: 1
+I: 270 [DL] Setup Pin Code: 20202021
+I: 273 [DL] Setup Discriminator: 3840 (0xF00)
+I: 278 [DL] Manufacturing Date: (not set)
+I: 281 [DL] Device Type: 65535 (0xFFFF)
 ```
 
-2. Scan BLE devices
+Run the following command to establish secure connection over Bluetooth LE, with the following assumptions for the Matter accessory device:
+ - The discriminator of the device is 3840
+ - The setup pin code of the device is 20202021
+ - The temporary Node ID is 1234
```

Comment:
Please put globally the values like these into single asterisks *...* whenever they are mentioned in the command description.

Reply:
Restyler forces me to change \*3840\* to \_3840\_, but I think this makes no difference for the user.